addOns/wappalyzer/src/main/javahelp/org/zaproxy/zap/extension/wappalyzer/resources/help/contents/automation.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Technology Detection Automation Framework Support
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Technology Detection Automation Framework Support</H1>
+This add-on supports the Automation Framework.
+<br><br>
+<H2>Job: wappalyzer</H2>
+The <code>wappalyzer</code> job is a data job. It does not have any configurable parameters.
+It provides technology detection data to other jobs via the <code>TechJobResultData</code> class.
+
+<H2>YAML</H2>
+
+<pre>
+  - type: wappalyzer             # Passive scan wait for the passive scanner to finish
```

Comment:
Maybe this is modelled based on another example but it just seems awkward to me.

If there's no good description for the job I think advice would be better handled as a note below or leading in.

Or maybe make sure the comment is clear that it's a note not a description? Just start it with `# Note:` and include the actual psw type/name?

Reply:
Sorry, this was a copy / paste issue. Removing the comment.